diff --git a/node_modules/prism-react-renderer/es/vendor/prism/prism-core.js b/node_modules/prism-react-renderer/es/vendor/prism/prism-core.js
index 1984434..2e26fe8 100755
--- a/node_modules/prism-react-renderer/es/vendor/prism/prism-core.js
+++ b/node_modules/prism-react-renderer/es/vendor/prism/prism-core.js
@@ -204,7 +204,7 @@ var Prism = function () {
           pattern = pattern.pattern || pattern; // Don’t cache length as it changes during the loop
 
           for (var i = index, pos = startPos; i < strarr.length; pos += strarr[i].length, ++i) {
-            var str = strarr[i];
+            var str = strarr[i] || ' ';
 
             if (strarr.length > text.length) {
               // Something went terribly wrong, ABORT, ABORT!
